package.json

```diff
@@ -66,7 +66,7 @@
     "cozy-doctypes": "1.72.2",
     "cozy-flags": "1.10.0",
     "cozy-logger": "1.6.0",
-    "cozy-ui": "35.39.0",
```

Comment:
This commit should not be `chore` since `chore` is reserved for build tool or something that is not published in production. Since cozy-ui is included in the production build, `chore` is not the right tag. 

Either `fix` if this upgrade brings a fix for Contacts, or `feat` if it brings a new feature or somethign else